The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9817516766/job/27108970358) [(plain)](https://github.com/rust-lang/rust/commit/2c14dd3b12d29749465a84429a4643da5c4e3934/checks/27108970358/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Getting action download info
Download action repository 'msys2/setup-msys2@v2.22.0' (SHA:cc11e9188b693c2b100158c3322424c4cc1dadea)
Download action repository 'actions/checkout@v4' (SHA:692973e3d937129bcbf40652eb9f2f61becf3332)
Download action repository 'actions/upload-artifact@v4' (SHA:0b2256b8c012f0828dc542b3febcab082c67f72b)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
COPY scripts/sccache.sh /scripts/
RUN sh /scripts/sccache.sh

COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt \
    && pip3 install virtualenv
COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---

#12 [5/8] COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
#12 DONE 0.0s

#13 [6/8] RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
#13 0.425   Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
#13 0.438 Collecting boolean-py==4.0
#13 0.442   Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
#13 0.454 Collecting chardet==5.1.0
---
#13 3.515 Building wheels for collected packages: reuse
#13 3.516   Building wheel for reuse (pyproject.toml): started
#13 3.847   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.848   Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=181117 sha256=f5f58750481f69515c2c0d1d503daf565e2565c370d07fc6aeb95fe3498b4269
#13 3.848   Stored in directory: /tmp/pip-ephem-wheel-cache-q88ndp9u/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
#13 3.851 Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
#13 3.875   Attempting uninstall: setuptools
#13 3.876     Found existing installation: setuptools 59.6.0
#13 3.877     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
---
#13 5.135   Downloading virtualenv-20.26.3-py3-none-any.whl (5.7 MB)
#13 5.203      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.7/5.7 MB 87.8 MB/s eta 0:00:00
#13 5.260 Collecting filelock<4,>=3.12.2
#13 5.264   Downloading filelock-3.15.4-py3-none-any.whl (16 kB)
#13 5.297 Collecting platformdirs<5,>=3.9.1
#13 5.302   Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
#13 5.322 Collecting distlib<1,>=0.3.7
#13 5.327   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 5.335      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 91.8 MB/s eta 0:00:00
#13 5.428 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 5.621 Successfully installed distlib-0.3.8 filelock-3.15.4 platformdirs-4.2.2 virtualenv-20.26.3
#13 DONE 5.7s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      219072 kB
DirectMap2M:     7120896 kB
DirectMap1G:    11534336 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/51917e2e69702e5752bce6a4f3bfd285d0f4ae39/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-51917e2e69702e5752bce6a4f3bfd285d0f4ae39-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished `release` profile [optimized] target(s) in 29.63s
##[endgroup]
fmt check
##[error]Diff in /checkout/library/alloc/src/boxed.rs at line 839:
     /// ```
     /// ```
     /// #![feature(allocator_api, new_uninit)]
+    ///
     /// use std::alloc::System;
     ///
     ///
     /// let mut values = Box::<[u32]>::try_new_uninit_slice(3, System)?;
##[error]Diff in /checkout/library/alloc/src/boxed.rs at line 856:
     /// ```
     #[unstable(feature = "allocator_api", issue = "32838")]
     #[inline]
-    pub fn try_new_uninit_slice_in(len: usize, alloc: A) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
+    pub fn try_new_uninit_slice_in(
+        len: usize,
+        alloc: A,
+    ) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
         let ptr = if T::IS_ZST || len == 0 {
         } else {
         } else {
##[error]Diff in /checkout/library/alloc/src/boxed.rs at line 879:
     /// ```
     /// ```
     /// #![feature(allocator_api, new_uninit)]
+    ///
     /// use std::alloc::System;
     ///
     ///
     /// let values = Box::<[u32]>::try_new_zeroed_slice(3, System)?;
##[error]Diff in /checkout/library/alloc/src/boxed.rs at line 892:
     /// [zeroed]: mem::MaybeUninit::zeroed
     #[unstable(feature = "allocator_api", issue = "32838")]
     #[inline]
-    pub fn try_new_zeroed_slice_in(len: usize, alloc: A) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
+    pub fn try_new_zeroed_slice_in(
+        len: usize,
+        alloc: A,
+    ) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
         let ptr = if T::IS_ZST || len == 0 {
         } else {
         } else {
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/library/std/src/os/netbsd/net.rs" "/checkout/library/std/src/os/netbsd/raw.rs" "/checkout/library/std/src/os/netbsd/fs.rs" "/checkout/library/alloc/tests/slice.rs" "/checkout/library/alloc/tests/arc.rs" "/checkout/library/alloc/tests/borrow.rs" "/checkout/library/alloc/tests/lib.rs" "/checkout/library/alloc/tests/vec_deque_alloc_error.rs" "/checkout/library/alloc/tests/btree_set_hash.rs" "/checkout/library/alloc/tests/autotraits.rs" "/checkout/library/alloc/tests/const_fns.rs" "/checkout/library/alloc/tests/thin_box.rs" "/checkout/library/alloc/tests/linked_list.rs" "/checkout/library/alloc/tests/fmt.rs" "/checkout/library/std/src/os/ios/mod.rs" "/checkout/library/std/src/os/ios/raw.rs" "/checkout/library/std/src/os/ios/fs.rs" "/checkout/library/std/src/os/openbsd/mod.rs" "/checkout/library/std/src/os/openbsd/raw.rs" "/checkout/library/std/src/os/openbsd/fs.rs" "/checkout/library/std/src/os/unix/process.rs" "/checkout/library/std/src/os/unix/thread.rs" "/checkout/library/std/src/os/unix/mod.rs" "/checkout/library/alloc/src/boxed.rs" "/checkout/library/std/src/os/unix/net/listener.rs" "/checkout/library/std/src/os/unix/net/mod.rs" "/checkout/library/std/src/os/unix/net/ancillary.rs" "/checkout/library/std/src/os/unix/net/addr.rs" "/checkout/library/std/src/os/unix/net/stream.rs" "/checkout/library/std/src/os/unix/net/tests.rs" "/checkout/library/std/src/os/unix/net/datagram.rs" "/checkout/library/std/src/os/unix/net/ucred/tests.rs" "/checkout/library/std/src/os/unix/net/ucred.rs" "/checkout/library/std/src/os/unix/fs/tests.rs" "/checkout/library/std/src/os/unix/io/mod.rs" "/checkout/library/std/src/os/unix/io/tests.rs" "/checkout/library/std/src/os/unix/ffi/os_str.rs" "/checkout/library/std/src/os/unix/ffi/mod.rs" "/checkout/library/std/src/os/unix/raw.rs" "/checkout/library/std/src/os/unix/fs.rs" "/checkout/library/std/src/os/raw/mod.rs" "/checkout/library/std/src/os/raw/tests.rs" "/checkout/library/std/src/os/hermit/mod.rs" "/checkout/library/std/src/os/hermit/io/mod.rs" "/checkout/library/std/src/os/hermit/io/net.rs" "/checkout/library/std/src/os/hermit/ffi.rs" "/checkout/library/std/src/os/android/mod.rs" "/checkout/library/std/src/os/android/net.rs" "/checkout/library/std/src/os/android/raw.rs" "/checkout/library/std/src/os/android/fs.rs" "/checkout/library/std/src/os/illumos/mod.rs" "/checkout/library/std/src/os/illumos/raw.rs" "/checkout/library/std/src/os/illumos/fs.rs" "/checkout/library/std/src/os/xous/mod.rs" "/checkout/library/std/src/os/xous/ffi/definitions.rs" "/checkout/library/std/src/os/xous/ffi/definitions/memoryflags.rs" "/checkout/library/std/src/os/xous/services/ticktimer.rs" "/checkout/library/std/src/os/xous/services/log.rs" "/checkout/library/std/src/os/xous/services/net.rs" "/checkout/library/std/src/os/xous/services/dns.rs" "/checkout/library/std/src/os/xous/services/systime.rs" "/checkout/library/std/src/os/xous/services.rs" "/checkout/library/std/src/os/xous/ffi.rs" "/checkout/library/std/src/os/netbsd/mod.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Sat Jul  6 07:19:22 UTC 2024
  network time: Sat, 06 Jul 2024 07:19:22 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/alloc/src/boxed.rs
+++ b/library/alloc/src/boxed.rs
@@ -842,7 +842,7 @@ impl<T, A: Allocator> Box<[T], A> {
     ///
     /// use std::alloc::System;
     ///
-    /// let mut values = Box::<[u32]>::try_new_uninit_slice_in(3, System)?;
+    /// let mut values = Box::<[u32], _>::try_new_uninit_slice_in(3, System)?;
     /// let values = unsafe {
     ///     // Deferred initialization:
     ///     values[0].as_mut_ptr().write(1);
@@ -885,7 +885,7 @@ impl<T, A: Allocator> Box<[T], A> {
     ///
     /// use std::alloc::System;
     ///
-    /// let values = Box::<[u32]>::try_new_zeroed_slice_in(3, System)?;
+    /// let values = Box::<[u32], _>::try_new_zeroed_slice_in(3, System)?;
     /// let values = unsafe { values.assume_init() };
     ///
     /// assert_eq!(*values, [0, 0, 0]);

--- a/library/alloc/src/boxed.rs
+++ b/library/alloc/src/boxed.rs
@@ -839,7 +839,7 @@ impl<T, A: Allocator> Box<[T], A> {
     ///
     /// ```
     /// #![feature(allocator_api, new_uninit)]
-    /// 
+    ///
     /// use std::alloc::System;
     ///
     /// let mut values = Box::<[u32]>::try_new_uninit_slice(3, System)?;
@@ -856,7 +856,10 @@ impl<T, A: Allocator> Box<[T], A> {
     /// ```
     #[unstable(feature = "allocator_api", issue = "32838")]
     #[inline]
-    pub fn try_new_uninit_slice_in(len: usize, alloc: A) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
+    pub fn try_new_uninit_slice_in(
+        len: usize,
+        alloc: A,
+    ) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
         let ptr = if T::IS_ZST || len == 0 {
             NonNull::dangling()
         } else {
@@ -879,7 +882,7 @@ impl<T, A: Allocator> Box<[T], A> {
     ///
     /// ```
     /// #![feature(allocator_api, new_uninit)]
-    /// 
+    ///
     /// use std::alloc::System;
     ///
     /// let values = Box::<[u32]>::try_new_zeroed_slice(3, System)?;
@@ -892,7 +895,10 @@ impl<T, A: Allocator> Box<[T], A> {
     /// [zeroed]: mem::MaybeUninit::zeroed
     #[unstable(feature = "allocator_api", issue = "32838")]
     #[inline]
-    pub fn try_new_zeroed_slice_in(len: usize, alloc: A) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
+    pub fn try_new_zeroed_slice_in(
+        len: usize,
+        alloc: A,
+    ) -> Result<Box<[mem::MaybeUninit<T>], A>, AllocError> {
         let ptr = if T::IS_ZST || len == 0 {
             NonNull::dangling()
         } else {

--- a/library/alloc/src/boxed.rs
+++ b/library/alloc/src/boxed.rs
@@ -842,7 +842,7 @@ impl<T, A: Allocator> Box<[T], A> {
     ///
     /// use std::alloc::System;
     ///
-    /// let mut values = Box::<[u32]>::try_new_uninit_slice(3, System)?;
+    /// let mut values = Box::<[u32]>::try_new_uninit_slice_in(3, System)?;
     /// let values = unsafe {
     ///     // Deferred initialization:
     ///     values[0].as_mut_ptr().write(1);
@@ -885,7 +885,7 @@ impl<T, A: Allocator> Box<[T], A> {
     ///
     /// use std::alloc::System;
     ///
-    /// let values = Box::<[u32]>::try_new_zeroed_slice(3, System)?;
+    /// let values = Box::<[u32]>::try_new_zeroed_slice_in(3, System)?;
     /// let values = unsafe { values.assume_init() };
     ///
     /// assert_eq!(*values, [0, 0, 0]);

